scripts/tracevis.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2021 ETH Zurich and University of Bologna.
+# Copyright 2023 ETH Zurich and University of Bologna.
```

Comment:
Can you please revert this change? The date in copyright headers should reflect the date the file was first published.

Reply:
Oh, yes sure. Good to know